src/patternfly/components/DataToolbar/docs/data-toolbar-intro.md

```diff
@@ -0,0 +1,45 @@
+Data toolbar relies on groups (`.pf-c-data-toolbar__group`) and items (`.pf-c-data-toolbar__item`), with default spacer values. Groups and items can be siblings and/or items can be nested within groups. Modifier selectors adjust spacing based on the type of group or item. Each modifier applies a unique CSS variable, therefore, the base spacer value for all elements can be customized and item/groups spacers can be themed individually. The default spacer value for items and groups is set to `--pf-global--spacer--md` or 16px.
+
+<br>
+### Default spacing for items and groups:
+| Class | CSS Variable | Computed Value |
+| -- | -- | -- |
+| `.pf-c-data-toolbar__item` | `--pf-c-data-toolbar__item--spacer` | `16px` |
+| `.pf-c-data-toolbar__group` | `--pf-c-data-toolbar__group--spacer` | `16px` |
+
+<br>
+### Data toolbar item types
+| Class | Applied to | Outcome |
+| -- | -- | -- |
+| `.pf-m-bulk-select` | `.pf-c-data-toolbar__item` | Modifies toolbar item spacing. Spacer value is set to `var(--pf-c-data-toolbar__item--m-bulk-select--spacer)`. |
+| `.pf-m-overflow-menu` | `.pf-c-data-toolbar__item` | Modifies toolbar item spacing. Spacer value is set to `var(--pf-c-data-toolbar__item--m-overflow-menu--spacer)`. |
+| `.pf-m-pagination` | `.pf-c-data-toolbar__item` | Modifies toolbar item spacing. Spacer value is set to `var(--pf-c-data-toolbar__item--m-pagination--spacer)`. |
+<br>
+<h3 class="Example_heading">Spacer system</h3>
```

Comment:
The indentation on this gets a little wonky in markdown. I replaced this through the "available breakpoints" line with this code and it looks better

```
<h3 class="Example_heading">Spacer system</h3>

**Spacer** - `.pf-m-spacer-{none, sm, md, lg}`.
- This modifier can be applied to a group or item.
- This modifier changes the spacer value for only the element to which it is applied.
- Responsive spacers can be used by appending `{-on-[breakpoint]}` to `.pf-m-spacer-{size}`. **Example:** `.pf-m-spacer-lg-on-xl`.

<br>
**Space items** - `.pf-m-space-items-{none, sm, md, lg}`.
- This modifier affects only groups.
- It changes the spacing of direct child items and can be overwritten with any other modifier.
- Responsive spacers can be used by appending `{-on-[breakpoint]}` to `.pf-m-space-items-{size}`. **Example:** `.pf-m-space-items-lg-on-xl`.

<br>
### Available breakpoints are: `-on-md, -on-lg, -on-xl, -on-2xl`.
```